ISSUE_TEMPLATE/membership.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Organization Membership Request
+about: Request membership in the OpenFeature Org
+title: 'REQUEST: New membership for <your-GH-handle>'
+labels: area/github-membership
+assignees: ''
+
+---
+
+<!-- Please remember to change the title of this issue by replacing
+ <your-GH-handle> with the actual GitHub handle -->
+
+### GitHub Username
+
+e.g. (at)example_user
+
+### Requirements
+
+- [ ] I have reviewed the community membership guidelines ADD LINK
```

Comment:
Add a link to the guidelines

Reply:
we need to first publish the contributor ladder